packages/dashboard/src/components/TfChainConnector.vue

```diff
@@ -465,6 +464,13 @@ export default class TfChainConnector extends Vue {
   public openAcceptTerms = false;
   public generatingAccount = false;
   public termsLoading = false;
+
+  public openTerms() {
+    this.openAcceptTerms = this.termsLoading = true;
+    setTimeout(() => {
+      this.termsLoading = false;
+    }, 2000);
```

Comment:
Why we used this timeout function?

Reply:
The iframe visibility is determined on termsLoading boolean, which changes once it gets loaded. This happens only once and after that, it gets stuck in the loading phase. I set the boolean outside the iframe itself and the issue was fixed.

Reply:
The timeout function for setting a delay so the progress circle would be shown for 2 seconds. The iframe itself doesn't need loading as it's already included in the dom but I didn't want to change the flow.